Documentation/using-the-client.md

```diff
@@ -104,6 +104,17 @@ Unit goodbye.service loaded on 85c0c595.../172.17.8.102
 
 If the unit does not exist when calling `start`, fleetctl will first search for a local unit file, submit it and schedule it.
 
+### Restart units
+
+Fleet currently doesn't have a dedicated `restart` command;
```

Comment:
Could you also put all links references at the bottom? https://github.com/coreos/docs/blob/master/STYLE.md#marking-down-the-link

I.e.
```
[use the `fleetctl ssh` subcommand][ssh-dynamically]
... ... ...
[ssh-dynamically]: #ssh-dynamically-to-host
EOF
```